sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/BeamSqlSeekableTable.java

```diff
@@ -28,8 +29,12 @@
  * FROM FACT_TABLE JOIN LOOKUP_TABLE ON ...}.
  */
 public interface BeamSqlSeekableTable extends Serializable {
-  /** prepare the instance. */
-  default void setUp() {}
+  /**
+   * prepare the instance
+   *
+   * @param joinSubsetType joining subset schema
+   */
+  default void setUp(Schema joinSubsetType) {}
```

Comment:
this could be a breaking change if other developer has implemented setUp() in their BeamSqlSeekableTable. Let's add this change to CHANGES.md

Reply:
added